hs_core/models.py

```diff
@@ -326,6 +328,7 @@ def page_permissions_page_processor(request, page):
         "users_json": users_json,
         "owners": owners,
         "self_access_level": self_access_level,
+        "can_copy": can_copy,
```

Comment:
I think self_access_level should be sufficient to know whether copy is allowed or not.

Reply:
That's what originally was happening with `{% if cm.raccess.public or self_access_level != 'None' %}` but it isn't sufficient because it displays the copy icon when a resource is discoverable without access levels.
![image](https://user-images.githubusercontent.com/35820390/78943222-9accd200-7a78-11ea-90df-81ccb5812c21.png)

Reply:
I think ```allow_resource_copy``` context variable can be used instead, so "can_copy" context variable most likely is not needed.